### DISCUSSION TITLE
统一管理DTK项目版本升级

### DISCUSSION THREAD

**Discussion by asterwyx** [linuxdeepin/developer-center#4042] [Show and tell | 成果展示]
DTK组准备统一管理所有DTK项目的版本，并且更换项目版本发布模式。具体事项如下：

- [x] 添加元文件，记录各个组件的版本
- [x] 添加工作流，批量发起版本升级PR以及处理其他工作事项
- [x] 添加Git submodule，让DTK仓库可以直接访问到各个DTK组件
- [ ] 更改发布模式，采用master分支大版本，建立维护分支发布小版本的发布模式
- [ ] 添加工作流，提供根据COMMIT_MSG自动cherry-pick相应的提交到维护分支

**Comment by zhoufan1989** [comment]
希望Vscode也能以插件形式管理和使用DTK开发套件。

**Reply by BLumia** [reply]
配好 CMake 和 C++ 开发的相关插件其实就可以了。
